src/java/org/apache/cassandra/config/DataRateSpec.java

```diff
@@ -252,6 +253,21 @@ public LongBytesPerSecondBound(long bytesPerSecond)
         {
             this(bytesPerSecond, BYTES_PER_SECOND);
         }
+
+        // this one should be used only for backward compatibility for stream_throughput_outbound and inter_dc_stream_throughput_outbound
+        // which were in megabits per second in 4.0. Do not start using it for any new properties
+        public static LongBytesPerSecondBound megabitsPerSecondInBytesPerSecond(long megabitsPerSecond)
+        {
+            final double BYTES_PER_MEGABIT = 125_000;
+            double bytesPerSecond = (double) megabitsPerSecond * BYTES_PER_MEGABIT;
```

Comment:
Why use double here? long should be fine
```suggestion
            final long BYTES_PER_MEGABIT = 125_000L;
            long bytesPerSecond = megabitsPerSecond * BYTES_PER_MEGABIT;
```

Reply:
It should work but I kept it consistent with bytesPerSecond which is double in the constructor.. which is the other question you had which I thinks falls back to weather we want to keep the mebibytes nested class here
@maedhroz any thoughts?

Reply:
I'm fine w/ changing to `long` we won't have fractional bytes, right?

Reply:
Depends on whether we keep the `IntMebibytesBound`. I think probably we don't but we would want `IntBytesBound` at some point. I guess that one we can add in trunk so people can use it. We do not expect new properties in 4.1 anymore? The question is what do we want in this ticket... remove also IntMebibytesBound and switch to long in DataRateSpec, but not in the DataRateUnit? And people need to be clear they need to use double for any new getters that will return rounded value in bigger than bytes unit....

Reply:
If there isn't anything that uses it, and we have no near-term idea of what might use it, I'd lean toward getting rid of `IntMebibytesBound`.